Document/0x05g-Testing-Network-Communication.md

```diff
@@ -302,44 +302,37 @@ The SSL pinning process should be implemented as described on the static analysi
 
 #### Overview
 
-[Provide a general description of the issue.]
+For sensitive applications, like banking applciations, OWASP MASVS introduces "Defense in Depth" verification level [1]. Critical operations (e.g. user enrollment, or account recovery) of such sensitive applications are the most attractive targets from attacker's perspective. This creates a need of implementing advanced security controls for such operations, like adding additional channels (e.g. SMS and e-mail) to confirm user's action. Additional channels may reduce a risk of many attacking scenarios (mainly phishing), but only when they are out of any security faults.
 
-#### Static Analysis
-
-[Describe how to assess this given either the source code or installer package (APK/IPA/etc.), but without running the app. Tailor this to the general situation (e.g., in some situations, having the decompiled classes is just as good as having the original source, in others it might make a bigger difference). If required, include a subsection about how to test with or without the original sources.]
-
-[Use the &lt;sup&gt; tag to reference external sources, e.g. Meyer's recipe for tomato soup<sup>[1]</sup>.]
 
-##### With Source Code
+#### Static Analysis
 
-##### Without Source Code
+Static analysis is not applicable to this point.
```

Comment:
Not sure about this: the topic might be to broad to address it all at once. I guess you can have multiple different scenarios that tell something about the strength of the additional channel. For this I guess you can for now say "static analysis TBD".